controllers/admin/AdminLinkWidgetController.php

```diff
@@ -160,6 +158,13 @@ public function renderForm()
                     'values' => $this->repository->getStaticPages(),
                     'desc' => $this->module->getTranslator()->trans('Please mark every page that you want to display in this block.', array(), 'Modules.LinkList')
                 ),
+                array(
+                    'type' => 'custom_pages',
+                    'label' => $this->module->getTranslator()->trans('Custom content', array(), 'Modules.LinkList'),
```

Comment:
`$this->getTranslator()` instead?
Also, @AlexEven could you check the added wording, please?